platforms/core-runtime/gradle-cli/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("gradlebuild.distribution.api-java")
+}
+
+description = "Implementation of the `gradle` command"
+
+dependencies {
+    api(libs.jsr305)
+    api(project(":base-services"))
+    api(project(":cli"))
+    api(project(":client-services"))
+    api(project(":concurrent"))
+    api(project(":daemon-protocol"))
+    api(project(":logging"))
+    api(project(":logging-api"))
+
+    // The client is able to run builds, so uses core and other projects
```

Comment:
Do we still intend to make this more of a "runtime" dependency, where we don't statically depend on the daemon, but instead load it dynamically? 

If so, can we amend the comment here?

Reply:
Eventually, but that's quite a deep change.